dissect/target/plugins/os/windows/defender.py

```diff
@@ -1,10 +1,13 @@
-from typing import Generator, Iterable, Any
 from datetime import datetime, timezone
+from io import BytesIO
+from pathlib import Path
+from typing import Any, BinaryIO, Generator, Iterable, Iterator
 
-from flow.record import Record
-
+import dissect.util.ts as ts
+from dissect.cstruct import Structure, cstruct
 from dissect.target import plugin
 from dissect.target.helpers.record import TargetRecordDescriptor
+from flow.record import Record
```

Comment:
Can you run isort? You can optionally add the following configuration to your `.vscode/settings.json`:
```
    "editor.codeActionsOnSave": {
        "source.organizeImports": true
    }
```